templates/results.hdbs

```diff
@@ -2,46 +2,76 @@
   {{#if this}}
```

Comment:
Yeah, this is getting kind of convoluted. Do you think it's necessary yet, or that it can wait until we add another data type?